backend/src/apiserver/template/tekton_template.go

```diff
@@ -331,8 +331,8 @@ func (t *Tekton) injectArchivalStep(workflow util.Workflow, artifactItemsJSON ma
 					t.getObjectFieldSelector("PIPELINERUN", "metadata.labels['tekton.dev/pipelineRun']"),
 					t.getObjectFieldSelector("PODNAME", "metadata.name"),
 					t.getObjectFieldSelector("NAMESPACE", "metadata.namespace"),
-					t.getSecretKeySelector("AWS_ACCESS_KEY_ID", "mlpipeline-minio-artifact", "accesskey"),
-					t.getSecretKeySelector("AWS_SECRET_ACCESS_KEY", "mlpipeline-minio-artifact", "secretkey"),
+					t.getEnvVar("AWS_ACCESS_KEY_ID", common.GetStringConfig(common.ObjectStoreAccessKey)),
```

Comment:
For the values in the `common` library, we should have some default value so that the default deployment and unit tests can fall back to the default behavior.